### ISSUE TITLE
No alternative for 'xdotool keydown'

### ISSUE THREAD

**Issue by vathanak-mao** [iberianpig/fusuma-plugin-sendkey#30]
For example, I would like to run 'xdotool keydown Alt'. Is there something like 'sendkeydown'?

**Comment by iberianpig** [comment]
I'm sorry for the delayed response.

Unfortunately, the Fusuma sendkey plugin does not have a function like 'sendkeydown' that replicates 'xdotool keydown Alt'. If you have a specific use case in mind, we can consider it further.

**Comment by mokshraj** [comment]
i would need it if want to hold the alt key if i want to create a alt tab switcher with it

**Comment by pota2dev** [comment]
Yeah, ydotool is not working in my machine. Need native "key up" and "key down" support inside this plugin.
